descriptors/ModuleDescriptor-template.json

```diff
@@ -574,7 +574,7 @@
     },
     {
       "id": "acquisition-methods",
-      "version": "1.0",
+      "version": "1.1",
```

Comment:
you not adding new fields or changing contract, so no need to increase version here

Reply:
reverted